101-SQL-Injection-Attack-Prevention/nested/microsoft.sql/servers.v12.0.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "sqlServerName": {
+        "type": "string"
+      },
+      "location": {
+        "type": "string"
+      },
+      "administratorLogin": {
+        "type": "string"
+      },
+      "administratorLoginPassword": {
+        "type": "securestring"
+      },
+      "tags": {
+        "type": "object"
+      }
+    },
+    "resources": [
+      {
+        "name": "[parameters('sqlServerName')]",
+        "type": "Microsoft.Sql/servers",
+        "apiVersion": "2015-05-01-preview",
+        "location": "[parameters('location')]",
+        "tags": "[parameters('tags')]",
+        "identity": {
+          "type": "SystemAssigned"
```

Comment:
what is this used for?

Reply:
This is used to create and assign an Azure AD principal for the resource

Reply:
I understand what it does, just not what it's used for - it's not used anywhere in the template and not output (so no way to actually get the id)